.github/workflows/ci.yaml

```diff
@@ -721,10 +713,10 @@ jobs:
       - name: Deploy Website to dotty-website
         uses: peaceiris/actions-gh-pages@v4
         with:
-          personal_token: ${{ env.DOTTY_WEBSITE_BOT_TOKEN }}
+          personal_token: ${{ secrets.DOTTYBOT_TOKEN }}
           publish_dir: docs/_site
-          external_repository: lampepfl/dotty-website
-          publish_branch: gh-pages
+          external_repository: scala/dotty.epfl.ch
+          publish_branch: main
```

Comment:
the repo seems to have `master` and `gh-pages`, so is `main` correct?

Reply:
Yes, I'm cloning as we speak and I will push one single branch named `main` out of `gh-pages` with the a version of the README that is in `master`. I can ping you when it's done.

Reply:
@lrytz done